src/streamlink/plugins/svtplay.py

```diff
@@ -12,7 +12,7 @@
 
 
 @pluginmatcher(re.compile(
-    r'https?://(?:www\.)?(?:svtplay|oppetarkiv)\.se(/(kanaler/)?)'
+    r'https?://(?:www\.)?(?:svtplay|oppetarkiv)\.se(/(kanaler/)?.*)'
```

Comment:
Appending `.*` to the regular expression doesn't do anything, [as `re.match` is being used for matching the input URL](https://github.com/streamlink/streamlink/blob/aa63c4eda9f47805aa7424c3cbeec55b8fe2dc27/src/streamlink/session.py#L339-L344), so it doesn't matter which comes afterwards unless there's an end-of-string anchor (`$`).

https://docs.python.org/3/library/re.html#re.match
> **re.match(pattern, string, flags=0)**
> If zero or more characters at the beginning of *string* match the regular expression pattern, return a corresponding match object.

These are the plugin URL tests, and as you can see, it matches URLs with arbitrary data:
https://github.com/streamlink/streamlink/blob/aa63c4eda9f47805aa7424c3cbeec55b8fe2dc27/tests/plugins/test_svtplay.py#L8-L24